sdk/tests/deqp/data/gles3/shaders/linkage.test

```diff
@@ -3492,9 +3494,9 @@ group uniform "Uniform linkage"
 	end
 
 	group block "Uniform blocks"
-		case precision_mismatch
+		case differing_precision
 			version 300 es
-			expect link_fail
+			expect build_successful
```

Comment:
It's markes as build_successful in native dEQP master branch:
https://android.googlesource.com/platform/external/deqp/+/master/data/gles3/shaders/linkage.test#3609

But it was removed from deqp-dev branch:
https://android.googlesource.com/platform/external/deqp/+/b8d0d0a842280c2594703ecce985563aebe1cde8